spec/models/course_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+require "rspec/collection_matchers"
+require "shoulda-matchers"
+
+RSpec.describe Course, type: :model do
+  let(:course) {FactoryBot.create :course}
+  subject {course}
```

Comment:
khai báo subject dùng làm gì e?

Reply:
Khai báo subject để chỉ định object sẽ được test, nhờ vậy, rspec có thể gọi đến các method trên object mà không cần phải gọi đến object. ví dụ:
- Không khai báo object : expect(course).to ...
- Có khai báo object : is_expected.to ...